net/iproute2.sh

```diff
@@ -380,6 +402,34 @@ _iproute2_policy_routing()
 
 iproute2_pre_start()
 {
+	local netns
+	_set_netns
+	if [ -n "${netns}" ] && [ -e /sys/class/net/"${IFACE}" ] ; then
+		local ip
+		ip=$(which ip)
```

Comment:
Are you sure this is portable? zsh seems to have a builtin `which` but neither bash, dash nor mksh do. Gentoo has `sys-apps/which` but we should not make netifrc depend on that package. Have you looked how other parts of netifrc try to find external programs?